packages/ember-glimmer/lib/index.js

```diff
@@ -1,3 +1,216 @@
+      There are {{blogPosts.length}} blog posts written by {{user.name}}.
+    </div>
+    {{#each blogPosts as |post|}}
+      <li>{{post.title}}</li>
+    {{/each}}
+  {{/with}}
+  ```
+
+  Without the `as` operator, it would be impossible to reference `user.name` in the example above.
+
+  NOTE: The alias should not reuse a name from the bound property path.
+
+  For example: `{{#with foo.bar as |foo|}}` is not supported because it attempts to alias using
+  the first part of the property path, `foo`. Instead, use `{{#with foo.bar as |baz|}}`.
+
+  @method with
+  @for Ember.Templates.helpers
+  @param {Object} options
+  @return {String} HTML string
+  @public
+ */
+
+
+/**
+  Execute the `debugger` statement in the current template's context.
+
+  ```handlebars
+  {{debugger}}
+  ```
+
+  When using the debugger helper you will have access to a `get` function. This
+  function retrieves values available in the context of the template.
+  For example, if you're wondering why a value `{{foo}}` isn't rendering as
+  expected within a template, you could place a `{{debugger}}` statement and,
+  when the `debugger;` breakpoint is hit, you can attempt to retrieve this value:
+
+  ```
+  > get('foo')
+  ```
+
+  `get` is also aware of keywords. So in this situation
+
+  ```handlebars
+  {{#each items as |item|}}
+    {{debugger}}
+  {{/each}}
+  ```
+
+  You'll be able to get values from the current item:
+
+  ```
+  > get('item.name')
+  ```
+
+  You can also access the context of the view to make sure it is the object that
+  you expect:
+
+  ```
+  > context
+  ```
+
+  @method debugger
+  @for Ember.Templates.helpers
+  @public
+ */
+
+/**
+  The `partial` helper renders another template without
+  changing the template context:
+
+  ```handlebars
+  {{foo}}
+  {{partial "nav"}}
+  ```
+
+  The above example template will render a template named
+  "-nav", which has the same context as the parent template
+  it's rendered into, so if the "-nav" template also referenced
+  `{{foo}}`, it would print the same thing as the `{{foo}}`
+  in the above example.
+
+  If a "-nav" template isn't found, the `partial` helper will
+  fall back to a template named "nav".
+
+  ### Bound template names
+
+  The parameter supplied to `partial` can also be a path
+  to a property containing a template name, e.g.:
+
+  ```handlebars
+  {{partial someTemplateName}}
+  ```
+
+  The above example will look up the value of `someTemplateName`
+  on the template context (e.g. a controller) and use that
+  value as the name of the template to render. If the resolved
+  value is falsy, nothing will be rendered. If `someTemplateName`
+  changes, the partial will be re-rendered using the new template
+  name.
+
+  @method partial
+  @for Ember.Templates.helpers
+  @param {String} partialName The name of the template to render minus the leading underscore.
+  @public
+*/
+
+/**
+ A method of subscribing to a stream which is safe for use with a non-stream
```

Comment:
No, it was only public behind a feature flag (was never exposed without flag enabled, and the flag was never enabled).  ✂️ all references to "streams"